packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -86,7 +86,64 @@ const getReadOnly = (
 }
 
 const ReflectionCard = (props: Props) => {
-  const {meeting, reflection, isClipped, openSpotlight, stackCount, showReactji, dataCy} = props
+  const {meetingRef, reflectionRef, isClipped, openSpotlight, stackCount, showReactji, dataCy} =
+    props
+  const reflection = useFragment(
+    graphql`
+      fragment ReflectionCard_reflection on RetroReflection {
+        ...ColorBadge_reflection
+        isViewerCreator
+        id
+        isEditing
+        meetingId
+        reflectionGroupId
+        promptId
+        content
+        reactjis {
+          ...ReactjiSection_reactjis
+          id
+          isViewerReactji
+        }
+        sortOrder
+      }
+    `,
+    reflectionRef
+  )
+  const meeting = useFragment(
+    graphql`
+      fragment ReflectionCard_meeting on RetrospectiveMeeting {
+        id
+        localPhase {
+          phaseType
+        }
+        localStage {
+          isComplete
+        }
+        meetingNumber
+        phases {
+          phaseType
+          stages {
+            id
+            isComplete
+          }
+        }
+        spotlightGroup {
+          id
+        }
+        spotlightSearchQuery
+        reflectionGroups {
```

Comment:
FYI: This will cause all reflection cards to re-render once something changes in an unrelated part. I thought this could be a problem, but they already re-render on master if there is a change.

Reply:
Can we do something about it as I need `reflectionGroups` for calculations inside logic?

Reply:
Actually this causes 2 renders each time something changes :-/

Reply:
I removed the `reflectionGroups` from `ReflectionCard` by passing the required data down from `GroupingKanban` but I am not able to see the difference in terms of render.
Could you please check, if it is what you mean?